src/us/kbase/workspace/test/workspace/WorkspaceTest.java

```diff
@@ -3950,21 +3987,11 @@ public void objectIDs() throws Exception {
 		testObjectIdentifier(goodWs, 0, "Object id must be > 0");
 		testObjectIdentifier(goodWs, 0, 1, "Object id must be > 0");
 		testObjectIdentifier(goodWs, 1, 0, "Object version must be > 0");
-		testCreate(goodWs, "f|o.A-1_2", null);
-		testCreate(goodWs, null, 1L);
-		testCreate(null, "boo", null, "wsi cannot be null");
-		testCreate(goodWs, TEXT256, null, "Object name exceeds the maximum length of 255");
-		testCreate(goodWs, null, null, "Must provide one and only one of object name (was: null) or id (was: null)");
-		testCreate(goodWs, "boo", 1L, "Must provide one and only one of object name (was: boo) or id (was: 1)");
-		testCreate(goodWs, "-1", null, "Object names cannot be integers: -1");
-		testCreate(goodWs, "15", null, "Object names cannot be integers: 15");
-		testCreateVer(goodWs, "boo", null, 1);
-		testCreateVer(goodWs, null, 1L, 1);
-		testCreateVer(goodWs, "boo", null, null);
-		testCreateVer(goodWs, null, 1L, null);
-		testCreateVer(goodWs, "boo", null, 0, "Object version must be > 0");
-		testCreateVer(goodWs, TEXT256, null, 1, "Object name exceeds the maximum length of 255");
-		testCreateVer(goodWs, null, 1L, 0, "Object version must be > 0");
+		testCreate(TEXT256, null, "Object name exceeds the maximum length of 255");
+		testCreate(null, null, "Must provide one and only one of object name (was: null) or id (was: null)");
+		testCreate("boo", 1L, "Must provide one and only one of object name (was: boo) or id (was: 1)");
+		testCreate("-1", null, "Object names cannot be integers: -1");
+		testCreate("15", null, "Object names cannot be integers: 15");
```

Comment:
These tests are now covered by the unit tests for ObjectIdentifier. The remaining tests cover ObjectIDNoWSNoVer